libdnf/dnf-repo.cpp

```diff
@@ -47,6 +47,11 @@
 #include "dnf-repo.hpp"
 #include "dnf-types.h"
 #include "dnf-utils.h"
+#include "libdnf/utils/File.hpp"
+
+#include <set>
```

Comment:
What about to move the includes above?

Reply:
Inserting standard header files after user header files helps to detect missing includes of standard header files in the user header files.